scripts/behavior.js

```diff
@@ -39,11 +39,11 @@ const COMMAND_FORBIDDEN = 4;
 /// Main bot behavior function
 ///
 function executeCommand(command, arguments, cooldown, message, settings, utils, callback){
-	
+

```

Comment:
hmm, my ide does that automatically, should I try to revert?

Reply:
no it's okay, this shouldn't have tabs but only 4 whitespaces imo